From cc4a255132cccc1c7c549dc70c227d0a94e6b895 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lawrence=20Vel=C3=A1zquez?= <larryv@macports.org>
Date: Tue, 3 Feb 2015 20:00:08 -0500
Subject: [PATCH 04/16] (MP) BuildSystem: Allow Xcode compiler override

If `GCC_VERSION` is not passed to the build, HandBrake's baked-in
compiler setting will be used.
---
 macosx/module.defs | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/macosx/module.defs b/macosx/module.defs
index d864b7f..773a8a1 100644
--- a/macosx/module.defs
+++ b/macosx/module.defs
@@ -53,6 +53,8 @@ MACOSX.XCODE = $(strip \
         $(MACOSX.configuration) \
         $(MACOSX.xcconfig) \
         \
+        $(if $(GCC_VERSION),GCC_VERSION='$(GCC_VERSION)') \
+        \
         SYMROOT='$(XCODE.symroot)' \
         CONFIGURATION_BUILD_DIR='$(XCODE.symroot)' \
         CONFIGURATION_TEMP_DIR='$(XCODE.symroot)' \
-- 
2.3.0

